utils/utils.go

```diff
@@ -255,6 +255,18 @@ func GeneratePasswd(uid, gid uint32, rootfs, rundir string) (string, error) {
 	if err != nil {
 		return "", errors.Wrapf(err, "unable to follow symlinks to passwd file")
 	}
+	info, err := os.Stat(originPasswdFile)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return "", nil
+		}
+		return "", errors.Wrapf(err, "unable to stat passwd file %s", originPasswdFile)
+	}
+	// Check if passwd file is world writable
+	if info.Mode().Perm()&(0002) != 0 {
```

Comment:
Should we also check separately for a group write permission?

Reply:
Changed to 0022